templates/instruc_template.forTasks.rst

```diff
@@ -0,0 +1,236 @@
+:orphan: true
+	 
+.. Based on: https://dmtn-030.lsst.io/v/DM-7096/index.html#task-topic-type, with learnings from the 4 sfp pages built in branch DM-8717
+
+########################
+Guidance for ExampleTask
+########################
+
+We give guidance here for developers on what to write into the
```

Comment:
Guess it's really covered by the way you rephrased it above, so no reason to load more words in when the essentials have been stated once.